integration-test/src/java/com/twitter/heron/integration_test/topology/serialization/resources/SerializationTopology.json

```diff
@@ -0,0 +1 @@
+[10]
```

Comment:
So [10] is the number of records we expect? Seems we'd also want to assert the data itself if possible.

How is IntegrationTestSpout used here, I only see CustomSpout in use?